cmd/scan.go

```diff
@@ -986,18 +989,24 @@ func (s *Scan) findORCreateProject() (*client.Project, error) {
 
 func statusMsg(s int) string {
 	switch s {
-	case jobStarting:
-		return "starting scan"
-	case jobRunning:
+	case eventStatusWaiting:
+		return "scan waiting"
+	case eventStatusStarting:
+		return "scan starting"
+	case eventStatusRunning:
 		return "scan running"
-	case jobAnalyzing:
+	case eventStatusRetrievingResults:
+		return "retrieving results"
+	case eventStatusAnalyzing:
 		return "analyzing scan results"
-	case jobNotifying:
+	case eventStatusNotifying:
 		return "setting notifications"
-	case jobFinished:
+	case eventStatusFinished:
 		return "scan finished"
+	case eventStatusFailed:
+		return "scan failed"
 	default:
-		return "unknown scan status"
+		return fmt.Sprintf("unknown status: %d", s)
 	}
 }
```

Comment:
Redundant usage. This method `statusMsg` not used anywhere. Can you check it please?

Reply:
52c79fc